packages/react/src/index.ts

```diff
@@ -24,7 +24,7 @@ export { signal, computed, batch, effect, Signal, type ReadonlySignal };
 const Empty = [] as const;
 const ReactElemType = Symbol.for("react.element"); // https://github.com/facebook/react/blob/346c7d4c43a0717302d446da9e7423a8e28d8996/packages/shared/ReactSymbols.js#L15
 const ReactMemoType = Symbol.for("react.memo"); // https://github.com/facebook/react/blob/346c7d4c43a0717302d446da9e7423a8e28d8996/packages/shared/ReactSymbols.js#L30
-const ProxyInstance = new Map<FunctionComponent<any>, FunctionComponent<any>>();
+const ProxyInstance = new WeakMap<FunctionComponent<any>, FunctionComponent<any>>();
```

Comment:
There are some runtimes that do not have WeakMap but do have Map.
Maybe we could do something like:
```ts
const ProxyInstance = new (typeof WeakMap === "function"
  ? WeakMap
  : Map
)();
```

Reply:
I think using `WeakMaps` is fine. They are supported even in IE11: https://caniuse.com/?search=JavaScript%20WeakMap